salesapp.js

```diff
@@ -96,6 +98,35 @@ function runAllStoreData() {
   }
 };
 
+function runStoreHourTotals(){
+  var total = 0;
```

Comment:
Total what? There might be a clearer way to name this variable, like `dailyTotal` or something.